nginx.conf

```diff
@@ -0,0 +1,31 @@
+events {
+	use epoll;
+	worker_connections 128;
+}
+
+http{
+
+server {
```

Comment:
Don't full understand why this is here but I don't know much about nginx.conf files.

Reply:
Whoops I was not supposed to add this file. This is the reverse proxy for my personal server. I'll delete it.